hunts/src/LinkCell.js

```diff
@@ -92,6 +128,19 @@ const GreenSheetsSvg = () => {
   );
 };
 
+const Clickable = ({ children, onClick, style, title }) => {
```

Comment:
I think you want to just replace this component with an `a` tag

Reply:
Sounds like I should change from `span` to `button` then?

Reply:
Personally I would just rip out the component and change `Clickable` to `button`, at that point all it's doing is applying `text-muted`

Reply:
Finally got around to updating this to `button`. Seems to work just as before. Thanks!